cursorless-talon/src/modifiers/scopes.py

```diff
@@ -4,6 +4,25 @@
 
 mod.list("cursorless_scope_type", desc="Supported scope types")
 mod.list("cursorless_scope_type_plural", desc="Supported plural scope types")
+
+mod.list(
```

Comment:
Since we're removing a bunch of captures it was easier to just move the scope lists here and remove those other files